packages/client/src/lib.js

```diff
@@ -266,6 +238,63 @@ class NFTStorage {
     }
   }
 
+  /**
+   * @template {API.TokenInput} T
+   * @param {API.Encoded<T, [[Blob, Blob]]>} input
```

Comment:
I think this is an error (which probably confused the type checker) `input` here supposed to be `T`, that is basically anything as long as it's compatible with `API.TokenInput`. 

Type `API.Encoded<T, [[Blob, Blob]]>` supposed to represent `T` in which all `Blobs` are substituted with `Blobs` which in theory should be the same `T`.